src/klein/_app.py

```diff
@@ -87,30 +127,32 @@ class Klein(object):
     _subroute_segments = 0
 
     def __init__(self):
+        # type: () -> None
         self._url_map = Map()
-        self._endpoints = {}
-        self._error_handlers = []
-        self._instance = None
-        self._boundAs = None
+        self._endpoints = {}       # type: Dict[Text, KleinRoute]
+        self._error_handlers = []  # type: List[KleinErrorHandler]
+        self._instance = None      # type: Klein # type: ignore[assignment]
```

Comment:
Not annotated as `Optional` because it shouldn't be set to `None`.

Reply:
It's fine for this to be `None` when the app isn't bound? i.e. if you have a module-level rather than a class-level app?

Reply:
OK I'll mark it optional